src/pudl/package_data/eia860/column_maps/generator_wind_existing.csv

```diff
@@ -0,0 +1,25 @@
+sector_name_eia,sector_name,sector_name,sector_name,sector_name,sector_name,sector_name,sector_name,sector_name,sector_name,sector_name
+sector_id_eia,sector,sector,sector,sector,sector,sector,sector,sector,sector,sector
+capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw,nameplate_capacity_mw
+summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw,summer_capacity_mw
+winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw,winter_capacity_mw
+generator_operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month,operating_month
+generator_operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year
+predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer,predominant_turbine_manufacturer
+predominant_turbine_model_num,predominant_turbine_model_number,predominant_turbine_model_number,predominant_turbine_model_number,predominant_turbine_model_number,predominant_turbine_model_number,predominant_turbine_model_number,predominant_turbine_model_number,predominant_turbine_model_number,predominant_turbine_model_number,predominant_turbine_model_number
+design_wind_speed_mph,design_wind_speed_mph,design_wind_speed_mph,design_wind_speed_mph,design_wind_speed_mph,design_wind_speed_mph,design_wind_speed_mph,design_wind_speed_mph,design_wind_speed_mph,design_wind_speed_mph,design_wind_speed_mph
+wind_quality_class,wind_quality_class,wind_quality_class,wind_quality_class,wind_quality_class,wind_quality_class,wind_quality_class,wind_quality_class,wind_quality_class,wind_quality_class,wind_quality_class
+turbine_hub_height_feet,turbine_hub_height_feet,turbine_hub_height_feet,turbine_hub_height_feet,turbine_hub_height_feet,turbine_hub_height_feet,turbine_hub_height_feet,turbine_hub_height_feet,turbine_hub_height_feet,turbine_hub_height_feet,turbine_hub_height_feet
+obstacle_num_faa,faa_obstacle_number,faa_obstacle_number,faa_obstacle_number,,,,,,,
+turbines_num,,,number_of_turbines,number_of_turbines,number_of_turbines,number_of_turbines,number_of_turbines,number_of_turbines,number_of_turbines,number_of_turbines
```

Comment:
just a note for the transform step `turbines_num` also shows up in the 2001-2012 current generators tab. So we'll want to make sure they end up in the same table.

Reply:
well have to figure that out when we get there. the mostly straightforward thing would be to harvest any column that is shared across the current generators tables and these tech-specifc tables. But yes if we make a wind-specific output table it could have a huge mostly chunk of records with mostly null data expect for ids + `turbines_num`